client/src/common/utils/helpers.js

```diff
@@ -0,0 +1,5 @@
+/**
+ *  Function checks if string contains only spaces, line breaks,
+ *  white spaces and returns true if so.
+ */
+export const whiteSpaceOnly = string => !string.replace(/\s/g, '').length;
```

Comment:
maybe just use: https://lodash.com/docs/4.17.11#trim and isEmpty?

Reply:
it doesn't work with lodash _isEmpty

Reply:
_isEmpty(_trim(value)) does not work?